app/src/main-process/log.ts

```diff
@@ -43,6 +43,14 @@ function initializeWinston(path: string): winston.LogMethod {
     maxFiles: MaxLogFiles,
   })
 
+  // The file logger handles errors when it can't write to an
+  // existing file but emits an error when attempting to create
+  // a file and failing (for example due to permissions or the
+  // disk being full). If logging fails that's not a big deal
+  // so we'll just suppress any error, besides, the console
+  // logger will likely still work.
+  fileLogger.on('error', _ => {})
```

Comment:
Coding style suggestion:

  - when no arguments are declared on a callback function, use `() => ` 
  - when parameters need to be declared but are unused (e.g. callback only needs the second parameter), use `_` as the argument name (or `_arg` if there are multiple args) as `tsc` can infer whether these are used 


<img width="515" src="https://user-images.githubusercontent.com/359239/54442040-aab29f00-471c-11e9-936c-4fb58d0d1091.png">

<img width="442" src="https://user-images.githubusercontent.com/359239/54442200-fb29fc80-471c-11e9-83a1-a9f2410840f4.png">


```suggestion
  fileLogger.on('error', () => {})
```